openweathermap-ios/openweathermap-ios/BookmarkLocation/Manager/BookmarkLocationManager.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import os.log
+
+class BookmarkLocationManager {
+
+    func save(bookmarkLocation: BookmarkLocation) throws -> [BookmarkLocation] {
+        var allBookmarks = try getAll()
+        let defaults = UserDefaults.standard
+        allBookmarks.append(bookmarkLocation)
+        let archiveData = try self.archiveData(allBookmarks)
+        defaults.set(archiveData, forKey: Bookmark.location.rawValue)
+        return allBookmarks
+    }
+
+    func getAll() throws -> [BookmarkLocation] {
```

Comment:
Why throws if you don't have any thrwable?

Reply:
Fixed in 212f90e